starknet_py/tests/e2e/docs/code_examples/test_contract.py

```diff
@@ -0,0 +1,106 @@
+# pylint: disable=unused-variable
+import pytest
+
+from starknet_py.contract import Contract
+from starknet_py.net import KeyPair
+from starknet_py.net.account.account import Account
+from starknet_py.net.gateway_client import GatewayClient
+from starknet_py.net.models import StarknetChainId
+from starknet_py.net.networks import TESTNET
+
+
+def test_init():
+    # docs-start: init
+    contract = Contract(
+        address=0x123,
+        abi=[
+            {
+                "inputs": [{"name": "amount", "type": "felt"}],
+                "name": "increase_balance",
+                "outputs": [],
+                "type": "function",
+            },
+        ],
+        provider=Account(
+            address=0x321,
+            client=GatewayClient(TESTNET),
+            key_pair=KeyPair(12, 34),
+            chain=StarknetChainId.TESTNET,
+        ),
+    )
+    # docs-end: init
+
+
+@pytest.mark.asyncio
+async def test_from_address(gateway_account, contract_address):
+    account = gateway_account
+    # docs-start: from_address
+    address = 1 or 0x1 or "0x1"
+    # docs-end: from_address
+    address = contract_address
+    # docs-start: from_address
+    contract = await Contract.from_address(address=address, provider=account)
+    # or if the contract is a proxy (read more about resolving proxies in the `Guide`)
```

Comment:
Wonder if `:ref:` would work here?

Reply:
Unfortunately not :/
<img width="360" alt="image" src="https://user-images.githubusercontent.com/61014013/215972328-2a26e80b-3cbb-4344-aa0d-adf6a17b45ac.png">